mesh/ncmesh.cpp

```diff
@@ -4261,7 +4261,7 @@ const CoarseFineTransformations& NCMesh::GetRefinementTransforms()
    if (!transforms.embeddings.Size())
    {
       transforms.Clear();
-      transforms.embeddings.SetSize(leaf_elements.Size());
+      transforms.embeddings.SetSize(leaf_elements.Size() - GetNumGhostElements());
```

Comment:
I think this can now (after `par-ncmesh-saving-dev`) be done more simply as `SetSize(NElements)`.